dependencies.dot

```diff
@@ -21,5 +21,7 @@ digraph {
   fv3core -> util
   physics -> util
   physics -> fv3core
+  fv3core -> shared
+  physics -> shared
```

Comment:
It may be confusing to have "shared" show up in this graph when the module is named "stencils".

Reply:
Changed to `stencils`